test/integration/garbage_collector_test.go

```diff
@@ -135,6 +138,8 @@ func setup(t *testing.T) (*garbagecollector.GarbageCollector, clientset.Interfac
 
 // This test simulates the cascading deletion.
 func TestCascadingDeletion(t *testing.T) {
+	glog.Infof("TestCascadingDeletion start\n")
+	flag.Set("v", "6")
```

Comment:
Can I keep these? It doesn't print out much because it's a small test but provides really useful information.